cypress/e2e/dashboard_spec.cy.js

```diff
@@ -58,7 +58,8 @@ describe('Rancid Tomatillos', () => {
   })
 
   it('Should reveal movie detail preview on hover', () => {
-    cy.get('.popup').should('be.hidden')
+    cy.viewport(1025, 700)
```

Comment:
I think it would be cool to look into types of assertions to make to test responsive design for different viewports. Could check the CSS properties possibly!